apps/docs/pages/guides/auth/auth-smtp.mdx

```diff
@@ -0,0 +1,38 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  id: 'auth-smtp',
+  title: 'Configure a Custom SMTP',
+  description: 'Moving towards production: Configuring a custom SMTP provider',
+}
+
+## Overview
+
+At present, you can trial the Supabase platform by sending up to four emails per hour via the built service. As you progress toward production, you may find yourself wanting for a custom SMTP service in order to increase your limits. A custom SMTP server will allow you to set your own cap on the number of emails sent per hour.
```

Comment:
Suggest adding some language around the 4 emails/hour not being guaranteed, and the default email service as a whole being offered on a best-effort basis